src/algebra/category/fgModule/limits.lean

```diff
@@ -28,7 +30,7 @@ universes v u
 open category_theory
 open category_theory.limits
 
-namespace FinVect
+namespace fgModule
 
 variables {J : Type} [small_category J] [fin_category J]
 variables {k : Type v} [field k]
```

Comment:
Does the content of this file hold for arbitrary (commutative) rings?

Reply:
Good question, but this stuff is low priority for me at the moment. I'm just trying to get these old PRs done, so I'd prefer not to include further generalisations here.

Reply:
Sure. 
maintainer merge